spring-cloud-sleuth-samples/spring-cloud-sleuth-sample-messaging/pom.xml

```diff
@@ -81,6 +81,10 @@
 			<artifactId>lombok</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>com.github.kristofa</groupId>
```

Comment:
I'll have to check out those - only there where we have the Zipkin server in samples brave should remain (only in test scope)